### PR TITLE
Add PMD side characters

### PR DESCRIPTION
**Pull request by DeathlyPlays**

### CODE REVIEW

**Comment by DeathlyPlays** [comment]
Credit to Mewth on this :P

**Comment by DeathlyPlays** [comment]
She loves PMD xD so when she learnt to code a bit from me this is her life xD

**Review by HoeenCoder — COMMENTED** [review]
Maybe we should possibly tweak the levels for some of these (the fully evolved ones)?

**Comment by ghost** [comment]
For balancing issues?

**Comment by DeathlyPlays** [comment]
Yeah ofc we could Hoeen just add ``level: (level),``

**Comment by DeathlyPlays** [comment]
But with it already setting everything to 25 as a default not much room to work with

**Comment by DeathlyPlays** [comment]
Other idea is to add the rarity thing like we did in Digimon as well

**Comment by HoeenCoder** [comment]
Its easy enough to modify the script to say: `If theres a level on the set, use that, otherwise use 25`

**Comment by DeathlyPlays** [comment]
Ik it actually would work without having the level in that changed Hoeen

**Comment by ghost** [comment]
Hmm, lets see HOW balanced this meta is with the new mons, IF certain mons become overpowered lower the level to 20 or something

**Review by ghost — COMMENTED** [review]
`gender: (variant === 1) ? 'M' : 'M',` Just looks wierd to define a definite gender. Why don't you use `gender: 'M',` ?

**Comment by Mystifi** [comment]
Thank you for mentioning that, @Eelektross, I was just about to do that myself, Just set it as `gender: 'M',` or `gender: 'F',` respectively.

File: mods/pmd/scripts.js [hunk 1]
```diff
@@ -499,6 +499,146 @@ exports.BattleScripts = {
...
+					atk: 252,
+					def: 8,
+				},
+				nature: 'Adamant',
+			},
+			'Guilds Chatot': {
+				species: 'Chatot',
+				ability: 'Tangled Feet',
+				item: '',
+				gender: (variant === 1) ? 'M' : 'M',
```
Comment (Eelektross) [inline mods/pmd/scripts.js]: I explained it in my review

**Comment by DeathlyPlays** [comment]
^

**Comment by HoeenCoder** [comment]
Why did you remove the variant code for mons that can be `M` or `F`?

**Comment by DeathlyPlays** [comment]
They automatically randomize gender if not specified @HoeenCoder

**Comment by HoeenCoder** [comment]
ah

**Comment by DeathlyPlays** [comment]
Tested and ready to go @HoeenCoder

**Comment by HoeenCoder** [comment]
I want to test the gender thing first, it dosent seem right to me but i could be wrong

**Comment by DeathlyPlays** [comment]
It works :P

**Pull request merged**